webapp/views/wpt-results.js

```diff
@@ -208,6 +208,9 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
                       <iron-icon class="bug" icon="bug-report"></iron-icon>
                     </a>
                   </template>
+                  <template is="dom-if" if="[[shouldDisplayTestLabel(node.path, metadataMap)]]">
+                    <iron-icon class="bug" src="/static/testlabel.svg" title="[[getTestLabel(node.path, metadataMap)]]"></iron-icon>
```

Comment:
I wonder what this will "look" like to screenreader users? Do we need to include "Label: " or something like that as descriptive text so that it doesn't just seems like it's part of the test name before? I don't have much experience with this stuff, but can you take a quick look into it?

Reply:
https://github.com/PolymerElements/iron-icon/issues/81 - looks like there is not way for `iron-icon` to support `alt` but it will appear the `title` text on the screen. I believe it will show title content  e.g. `interop123`. What if we name it to `label:interop123` for the title?

Reply:
Yeah, `label: interop-2022` when there's a single label, or `labels: topic1, topic2` when there are multiple sounds good. I've tried reading those out with Chrome's text-to-speech and the result is intelligible I think.

Reply:
Done